README.md

```diff
@@ -110,84 +110,105 @@ _Revisiting Class Cohesion: An empirical investigation on several systems_,<br/>
 Journal of Object Technology, vol.7, no.6, 2008,
 [PDF](http://www.jot.fm/issues/issue_2008_07/article1.pdf).
 
-Here is a list of metrics we already implement:
+### Here is a list of metrics we already implement:
```

Comment:
@victornoel I believe it should be `we've already implemented`

Reply:
@vatavuk both are correct english, but I will change, it is same the for me :)